packages/custom-functions-metadata/test/cases/input-parameter-types-cellvalue/expected.json

```diff
@@ -0,0 +1,205 @@
+{
+    "allowCustomDataForDataTypeAny": true,
+    "functions": [
+        {
+            "description": "Test Excel.CellValue parameter",
```

Comment:
There is currently only one successful case which is for Excel.CellValue. Really, there should be a case for each type.

Reply:
Sorry, could you clarify your comment? I have all the supported subtypes of CellValue here. What other type do you mean?

Reply:
Each of the *CellValue types.

Reply:
I hope I'm not missing something but I have all the *CellValue types here? If one of them fails the whole test will fail.